frontend/src/main/scala/bloop/Bloop.scala

```diff
@@ -52,4 +56,11 @@ object Bloop {
         run(state)
     }
   }
+
+  private def consoleReader(): ConsoleReader = {
```

Comment:
I think it would be better to inline this where `reader` is used, since it's only used once.